core/src/test/resources/sql/hep.iq

```diff
@@ -38,11 +38,11 @@
 # 5. Duplicate operations on the same rule: last operation wins
 #    Example: "+Rule1,+Rule1,-Rule1" results in rule being removed
 # 6. Execution order:
-#      [HEP Program] (HepPlanner, configurable rules)
+#      [HEP Program] (HepPlanner, configurable CoreRules.* rules)
 #            ↓
-#      [Volcano Program] (VolcanoPlanner, configurable rules)
+#      [Volcano Program] (VolcanoPlanner, configurable EnumerableRules.* rules)
 #            ↓
-#      [Calc Program] (HepPlanner, fixed rules)
+#      [Calc Program] (HepPlanner, fixed RelOptRules.CALC_RULES)
```

Comment:
Is this correct?
what prevents someone from adding a CALC_RULES to the VolcanoPlanner?
What you are listing here is the default content of the rules

Reply:
I mimicked the usage pattern of the Program.standard method and ultimately called Programs.calc.

Reply:
I am talking about what the comment documents.
What does `[Calc Program] (HepPlanner, fixed RelOptRules.CALC_RULES)` mean?
It seems to me that it says that there is a program executed by the HepPlanner which is initially seeded with the CALC_RULES.
If this is correct, you should say it explicitly in English.

Reply:
I have revised the arrow-style instructions into descriptive sentences. Could you please review if this is acceptable?